backend/src/main/java/com/example/backend/risk/example

```diff
@@ -5,7 +5,7 @@
         "riskName": "Test Name",
```

Comment:
Static and resource files should be placed under resources. This file also does not seem to be used anywhere. Please remove this!

Reply:
Fair enough
_Deleted_